packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/aspnet/remotesignout.md

```diff
@@ -0,0 +1,72 @@
+Open the controller where you handle the sign-out process and update the `SignOut` action. This time you need to also sign the user out of the Okta OIDC middleware with `OktaDefaults.MvcAuthenticationType`:
+
+```csharp
+public class AccountController : Controller
+{
+    [HttpPost]
+    public ActionResult SignOut()
+    {
+        if (HttpContext.User.Identity.IsAuthenticated)
+        {
+            HttpContext.GetOwinContext().Authentication.SignOut(
+                CookieAuthenticationDefaults.AuthenticationType,
+                OktaDefaults.MvcAuthenticationType);
+        }
+
+        return RedirectToAction("Index", "Home");
+    }
+}
+```
+
+Update your using statements to import the following namespaces:
+
+```csharp
+using Microsoft.Owin.Security.Cookies;
+using Okta.AspNet;
+```
+
+After you sign users out of your app and out of Okta, you can also redirect users to a specific location after signing out. You need to whitelist the post sign-out URL in your Okta Application settings.
```

Comment:
After you sign users out of your app and out of Okta, you can also redirect users to a specific location. You need to whitelist the post sign-out URL in your Okta Application settings.